src/library/scala/collection/mutable/UnrolledBuffer.scala

```diff
@@ -237,8 +237,13 @@ object UnrolledBuffer extends StrictOptimizedClassTagSeqFactory[UnrolledBuffer]
 
   def newBuilder[A : ClassTag]: UnrolledBuffer[A] = new UnrolledBuffer[A]
 
-  val waterline = 50
-  val waterlineDelim = 100    // TODO -- fix this name!  It's a denominator, not a delimiter.  (But it's part of the API so we can't just change it.)
+  final val waterline: Int = 50
+
+  final val waterlineDenom: Int = 100
+
+  @deprecated("Use waterlineDenom instead.", "2.13.0")
+  final val waterlineDelim: Int = waterlineDenom
```

Comment:
Is there any reason why this has to be a `val`? A `def` doesn't require an extra field and keeps serialization stability.

Reply:
Not that I'm aware of, I can change that 👍

Reply:
@szeiger I am assuming you meant to keep waterline and waterlineDelim as a `val` but set waterlineDenom as a def, that's what I've done now.

Reply:
I think breakage is less likely if the def references the val, not the other way around.  On the other hand, is it a problem to make everything a `def`?  There's no reason any of this should be serialized.